docs/docusaurus/src/css/tabs.scss

```diff
@@ -0,0 +1,33 @@
+.tabs-container .margin-vert--md{
+  margin-top: 0 !important;
+}
+
+div[role="tabpanel"]{
+  --ifm-code-background: var(--ifm-alert-background-color-highlight);
+  border-radius: 0;
+  box-shadow:none;
+  border-top: 0;
+  padding: 0;
+}
```

Comment:
Same with this one, I noticed you moved these two but I can't tell where they're being applied